components/layout_2020/table/layout.rs

```diff
@@ -450,7 +456,7 @@ impl Table {
         };
 
         let sizes = cell.inline_content_sizes(layout_context, writing_mode);
-        sizes.map(|size| size / cell.colspan as f32)
+        sizes.map(|size| Au::from_f32_px(Au::to_f32_px(size) / cell.colspan as f32))
```

Comment:
I think you can avoid the conversion in this case since size is already `Au` and we are just scaling by `colspan`.

Reply:
I am not sure if we can completely avoid conversion here. `size` is of type `Au`, while `cell.colspan` is a `usize`. Conversion seems necessary to ensure type compatibility for the division operation; otherwise, the compiler complains. However, I think it's better to perform the conversion on `cell.colspan` instead of `size`.

If you agree, I will go ahead and do that

Reply:
I see, updated in [23aef62](https://github.com/servo/servo/pull/31135/commits/23aef62e1323912904edce349aaae0e3e2149267). thanks :)